borg/archiver.py

```diff
@@ -129,6 +129,11 @@ def do_change_passphrase(self, args):
 
     def do_create(self, args):
         """Create new archive"""
+        matcher = PatternMatcher()
+        matcher.fallback = True
```

Comment:
An earlier version of `PatternMatcher` only supported booleans and was much more complicated. The current implementation is simpler and supports any fallback value type. For this reason the default is `None` and needs to be overwritten here.

I've changed `PatternMatcher` to directly expose `fallback` instead of doing so via a property as it's no longer necessary with the simplified version